# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1296510103 18000
#      Mon Jan 31 16:41:43 2011 -0500
# Node ID 126d8fe48a1200f30b83dd871a514602cd9fb418
# Parent  88fa9e9efdeae51932110c2b03e12f9cd967aea0
Fix warning about GOTO reference in arpack code

diff --git a/libcruft/ChangeLog b/libcruft/ChangeLog
--- a/libcruft/ChangeLog
+++ b/libcruft/ChangeLog
@@ -1,8 +1,13 @@
+2011-01-31  Rik  <octave@nomad.inbox5.com>
+
+	* arpack/src/dseupd.f, arpack/src/sseupd.f: Change GOTO target
+	to eliminate warning about landing on end if.
+
 2011-01-31  John W. Eaton  <jwe@octave.org>
 
 	* arpack/module.mk (EXTRA_DIST): Include arpack/module.mk in the
 	list.  Remove arpack/src/module.mk and arpacck/util/module.mk
 	from the list.
 
 2011-01-31  John W. Eaton  <jwe@octave.org>
 
diff --git a/libcruft/arpack/src/dseupd.f b/libcruft/arpack/src/dseupd.f
--- a/libcruft/arpack/src/dseupd.f
+++ b/libcruft/arpack/src/dseupd.f
@@ -604,19 +604,19 @@ c
      &                    workl(iq+ncv*(rghtptr-1)), 1)
                leftptr = leftptr + 1
                rghtptr = rghtptr - 1
 c
             end if
 c
             if (leftptr .lt. rghtptr) go to 20
 c
- 30      end if
+         end if
 c
-         if (msglvl .gt. 2) then
+ 30      if (msglvl .gt. 2) then
              call dvout  (logfil, ncv, workl(ihd), ndigit,
      &       '_seupd: The eigenvalues of H--reordered')
          end if
 c
 c        %----------------------------------------%
 c        | Load the converged Ritz values into D. |
 c        %----------------------------------------%
 c
diff --git a/libcruft/arpack/src/sseupd.f b/libcruft/arpack/src/sseupd.f
--- a/libcruft/arpack/src/sseupd.f
+++ b/libcruft/arpack/src/sseupd.f
@@ -604,19 +604,19 @@ c
      &                    workl(iq+ncv*(rghtptr-1)), 1)
                leftptr = leftptr + 1
                rghtptr = rghtptr - 1
 c
             end if
 c
             if (leftptr .lt. rghtptr) go to 20
 c
- 30      end if
+         end if
 c
-         if (msglvl .gt. 2) then
+ 30      if (msglvl .gt. 2) then
              call svout (logfil, ncv, workl(ihd), ndigit,
      &       '_seupd: The eigenvalues of H--reordered')
          end if
 c
 c        %----------------------------------------%
 c        | Load the converged Ritz values into D. |
 c        %----------------------------------------%
 c
